handlers/messagebird/handler.go

```diff
@@ -62,14 +62,16 @@ var statusMapping = map[string]courier.MsgStatus{
 	"expired":         courier.MsgStatusFailed,
 }
 
-type ReceivedMessage struct {
```

Comment:
Why not keep expecting the JSON format for this handler?

Reply:
the default webhook on MB is urlencoded, we were using a internal flow with them that turned it to json (among other things) before we found out they charge extra for these.

Reply:
Changing that will require you update each existing callback to use the default urlencoded again, I imagine those are now set to use JSON when submitting

Reply:
surprisingly sending with MB is either json or urlencoded, so that doesn't need to change. Since we only use them for shortcodes, we have a patch of our current courier with some of these changes just so we could move back to their default webhook to cut our SMS costs

Reply:
I think we can keep expecting JSON, and make that clear on the config page https://github.com/nyaruka/rapidpro/blob/main/temba/channels/types/messagebird/type.py#L35 that the callback should be configured with JSON

Reply:
Let's take a look and see if we have customers using messagebird that way... it would be nice to support whatever the default setup is for this channel.

Reply:
its basically like the difference between using the twilio webhook, or using a twilio function to convert the message to json. there's a cost associated that they don't make clear. the actual webhook from the number is always form encoded, you have to manually set up a flow (function) to convert. We did that before we knew there was a cost per run of that flow.

Reply:
Ok we have no channels of this type in prod probably because the type has always been beta gated. Since the default way to connect this channel sends urlencoded params, that's what we should support. And when we've merged this and tested it.. let's un- beta gate it.